src/vs/platform/actionWidget/browser/actionList.ts

```diff
@@ -32,6 +32,7 @@ export interface IActionListItem<T> {
 	readonly group?: { kind?: any; icon?: ThemeIcon; title: string };
 	readonly disabled?: boolean;
 	readonly label?: string;
+	readonly hasEdits?: boolean;
```

Comment:
How about `canPreview` instead? This aligns with name used in the bulkEditService